api/__test-data__/events.js

```diff
@@ -1,7 +1,7 @@
 const { faker } = require('@faker-js/faker');
 
 const eventTypes = ['Internal', 'External'];
-const durations = ['1 hour', '2 hours', '4 hours 30 minutes', '1 day'];
+const durations = [60, 120, 360, 480];
```

Comment:
Should we add a comment explaining these - it's obvious now but might appear as magic numbers later!

Reply:
Good shout, thanks -- have made that clearer now 👍